applications/fluid_structure_interaction/perpendicular_flap/fluid-exadg/input.json

```diff
@@ -0,0 +1,23 @@
+{
+    "General": {
+        "Precision": "double",
+        "Dim": "2",
+        "IsTest": "false"
+    },
+    "SpatialResolutionFluid": {
+        "Degree": "2",
+        "RefineSpace": "1"
+    },
+    "preciceConfiguration": {
+        "preciceConfigFile": "../precice-config.xml",
+        "Physics": "Fluid",
+        "ParticipantName": "Fluid",
+        "ReadMeshName": "Fluid-Mesh-read",
+        "WriteMeshName": "Fluid-Mesh-write",
+        "ALEMeshName": "ALE-Mesh",
+        "WriteDataSpecification": "values_on_q_points",
+        "DisplacementDataName": "Displacement",
+        "VelocityDataName": "Velocity",
+        "StressDataName": "Stress"
+    }
```

Comment:
shouldn't there be parameters for writing output? (the default value should be "false", but it would nevertheless make sense to include this section here, so that the user only has to switch "false" to "true".)